### PR TITLE
Feature - Convert Template chooser Dialog to use UI-React

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Description
I have used some hardcoded width to make the dialog look alike. Please suggest me a way to make this better.
We need to have list component in ui-react.

## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/components/templateChooserModal.tsx [hunk 1]
```diff
@@ -125,75 +52,76 @@ export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.R
...
-                          primary={template.name}
-                          secondary={template.description}
-                        />
-                        <div className={classes.templateButtons}>
+                    <ListItem key={template.templateId} divider={true}>
+                      <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}>
+                        <ListItemAvatar>
+                          <img width='100px' src={template.imageUrl} />
+                        </ListItemAvatar>
+                        <Box maxWidth='340px'>
```
Comment (krishan711) [inline src/packages/console/src/components/templateChooserModal.tsx]: why is this size here?

File: src/packages/console/src/components/templateChooserModal.tsx [hunk 2]
```diff
@@ -125,75 +52,76 @@ export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.R
...
-                      <div className={classes.templateContent}>
-                        <ListItemText
-                          primary={template.name}
-                          secondary={template.description}
-                        />
-                        <div className={classes.templateButtons}>
+                    <ListItem key={template.templateId} divider={true}>
+                      <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}>
+                        <ListItemAvatar>
+                          <img width='100px' src={template.imageUrl} />
```
Comment (krishan711) [inline src/packages/console/src/components/templateChooserModal.tsx]: why do you need the size?

File: src/packages/console/src/components/templateChooserModal.tsx (hunk 1 above)
Comment (MoiyadKayda) [reply src/packages/console/src/components/templateChooserModal.tsx]: Without giving it a width, the text would overflow and I was able to contain the text in a box

File: src/packages/console/src/components/templateChooserModal.tsx (hunk 2 above)
Comment (MoiyadKayda) [reply src/packages/console/src/components/templateChooserModal.tsx]: this was in the css class assigned to it before

Comment (MoiyadKayda) [reply src/packages/console/src/components/templateChooserModal.tsx]: So I just gave the width in the image tag

File: src/packages/console/src/components/templateChooserModal.tsx [hunk 3]
```diff
@@ -1,96 +1,22 @@
...
-  },
-}));
+import { Dialog } from './dialog';
 
 export interface ITemplateChooserModalProps {
   isOpen: boolean;
   onChooseTemplateClicked: (template: Template) => void;
 }
 
 export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.ReactElement => {
```
Comment (krishan711) [inline src/packages/console/src/components/templateChooserModal.tsx:19]: pls rename to Dialog (including the file name)

File: src/packages/console/src/components/templateChooserModal.tsx [hunk 4]
```diff
@@ -125,75 +51,76 @@ export const TemplateChooserModal = (props: ITemplateChooserModalProps): React.R
   };
 
   return (
-    <Modal
-      open={props.isOpen}
+    <Dialog
+      isOpen={props.isOpen}
+      maxWidth='75%'
+      onCloseClicked={() => false}
```
Comment (krishan711) [inline src/packages/console/src/components/templateChooserModal.tsx]: you should pass this out to a prop that is also onCloseClicked (like the other dialogs)

File: src/packages/console/src/components/templateChooserModal.tsx (hunk 1 above)
Comment (krishan711) [reply src/packages/console/src/components/templateChooserModal.tsx]: this doesnt seem right, can you try to replace the `ListItemText` with ui-react `Text` please (you will need to put two in a vertical stack fo the name and description separately)

**Review by krishan711 — APPROVED** [review]

**Pull request merged**